### ISSUE TITLE
"unable to ask for SMS password" when tty is missing

### ISSUE THREAD

**Issue by chilek** [jwilk/mbank-cli#82]
I've configured mbank-cli to use mobile app authorization and registered its instance using:
```bash
mbank-cli register-device
```
When I launch mbank-cli manually it does its job as it should, but when it is launched from cronie I've got following output:
```
mbank-cli: unable to ask for SMS password
```
Quite strange, because I don't event use sms password for authorization! What's going on?

**Comment by chilek** [comment]
I reply for myself - probable reason is fragment of `/etc/crontab` file:
```
HOME=/
```
I've added correct HOME value assumption in my shell wrapper script.

**Comment by chilek** [comment]
Unfortunately that was not the reason of my problem.

**Comment by chilek** [comment]
As summary:
1. When we launch script manually we don't have any interaction needs:
```sh
$ ./mbank-cli activate-profile business
$
```
2. When it's launched from cronie we get:
```
mbank-cli: unable to ask for SMS password
```

**Comment by chilek** [comment]
I've added `--verbose` command line parameter to mbank-cli launch from cronie. I'm waiting for new reports.

**Comment by jwilk** [comment]
There's a sanity check that prevents logging in if stdin is not a tty.
This made sense back when SMS 2FA whas the only option, because without tty there would be no way to complete the login process.

Delete this line a temporary work-around:
https://github.com/jwilk/mbank-cli/blob/f3971e0a3c81165cf745c8ab3f4c66f897e282ce/mbank-cli#L1391

**Comment by chilek** [comment]
thanks, I changed it and wait for next try.

**Comment by chilek** [comment]
I looks like helped!

**Comment by jwilk** [comment]
Fixed in 5f2a8bf97da4509650d7d27c1f07a4de6fdc65af.

I'll make a new bug-fix release later today.

**Comment by chilek** [comment]
pulled before the moment from master and now it's testing. Thanks!

**Comment by jwilk** [comment]
Fixed in [mbank-cli 2.2.1](https://github.com/jwilk/mbank-cli/releases/tag/2.2.1).
